src/factory/servo/servoPort.ts

```diff
@@ -6,15 +6,24 @@ import { delay } from '../../utils/delay'
 export const servoPort = (port: SerialPort) => {
   return (pin: number) => {
     return {
-      rotate: async (angle: number) => {
+      setAngle: async (angle: number,recentAngle: number | null) => {
```

Comment:
It is difficult for the user to enter the current angle.
Nor can it be obtained from the Arduino.
The current angle is the responsibility of the user, so please remove the second argument.